src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static Ball getRandomBall() {
+        int randomNumber = new Random().nextInt(101);
+        Ball ball = new Ball(ColorSupplier.getRandomColor(), randomNumber);
```

Comment:
same lets make Random and ColorSupplier as class level variables, not to create new instances every time we call getRandomBall() method. And lets move 101 to constant https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names